app/controllers/users_controller.rb

```diff
@@ -1,6 +1,14 @@
 class UsersController < ApplicationController
   before_action :authenticate
 
+  def show
+    @user = User.where(id: current_user).first
+
+    @events_owner = Event.where(owner_id: current_user).where(Event.arel_table[:start_time].gteq Time.zone.now).order(:start_time)
+
+    @events_part = Event.where(Event.arel_table[:start_time].gteq Time.zone.now).joins(:tickets).merge(Ticket.where(user_id: current_user)).order(:start_time)
```

Comment:
__[rubocop]__
* Style/NestedParenthesizedCalls: Add parentheses to nested method call `Event.arel_table[:start_time].gteq Time.zone.now`.
* Metrics/LineLength: Line is too long. [158/100] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)